remove_virtual_server.rb

```diff
@@ -1,7 +1,13 @@
 Backups::Plugin.hook helpers: %i[client_helper] do
   def call(virtual_server)
     virtual_server.metadata[:veeam_related_job_ids]&.each do |job_id|
-      api_put("jobs/#{job_id}", disable_schedule_params.to_xml)
+      begin
+          api_get("jobs/#{job_id}")
+      rescue => err
+          # Something went wrong while matching Veeam Job for the virtual server
+      else
+          api_put("jobs/#{job_id}", disable_schedule_params.to_xml)
+      end
```

Comment:
```ruby
begin
  api_get("jobs/#{job_id}")

  api_put("jobs/#{job_id}", disable_schedule_params.to_xml)
rescue RestClient::BadRequest
  # do nothing
end
```

Reply:
Done.

Reply:
But in case when `api_put` fails, you will rescue that request and won't do nothing. Is that expected behavior?

Reply:
Yep, if request fails, we just do nothing at Veeam side, and disable the resource for VM in OnApp.